lib/Dialect/SV/Transforms/RTLCleanup.cpp

```diff
@@ -137,10 +137,12 @@ void RTLCleanupPass::runOnGraphRegion(Region &region, bool shallow) {
     // Recursively process any regions in the op before we visit it.
     if (!shallow && op.getNumRegions() != 0)
       runOnRegionsInOp(op);
-
-    // Merge alwaysff operations by hashing them to check to see if we've
-    // already encountered one.  If so, merge them and reprocess the body.
-    if (auto alwaysOp = dyn_cast<sv::AlwaysFFOp>(op)) {
```

Comment:
alwaysOp is an Operation, so why not just do a ```if (isa<sv::AlwaysOp>(op) || isa<sv::AlwaysFFOp>(op))```.

Reply:
Done